src/manage_sql_secinfo.c

```diff
@@ -5135,7 +5135,7 @@ rebuild_scap (const char *type)
 {
   int updated_scap_ovaldefs, updated_scap_cpes, updated_scap_cves;
 
-  updated_scap_ovaldefs = 0;
+  // updated_scap_ovaldefs = 0; // initial value currently unused
```

Comment:
I know this might be used in the future but I think you should remove the line for now because it's confusing.  The initial value is always set below.

Reply:
This has been made obsolete by the refactoring to get rid of the duplication and inconsistencies you mentioned in your other comment.